content/docs/atom-guide/dev-guide/troubleshoot.md

```diff
@@ -63,4 +63,27 @@ You can enable validation in your RHI device to output additional debugging info
 
 - `--rhi-device-validation="enable"`: Enables basic driver validation.
 - `--rhi-device-validation="verbose"`: Enables basic driver validation and additional reporting.
-- `--rhi-device-validation="gpu"`: Enables GPU-based validation (GBV). This validation mode runs slower, but catches issues that the `enable` and `verbose` modes may not catch. Note that this mode may produce false positives.
\ No newline at end of file
+- `--rhi-device-validation="gpu"`: Enables GPU-based validation (GBV). This validation mode runs slower, but catches issues that the `enable` and `verbose` modes may not catch. Note that this mode may produce false positives.
+
+
+## DRED logs (Windows DirectX Only)
+
+When a TDR (Timeout Delay Recovery) crash occurs while using the DX12 backend specifically, [Device Removed Extended Data](https://microsoft.github.io/DirectX-Specs/d3d/DeviceRemovedExtendedData.html) (aka "DRED") breadcrumbs
+are written to a timestamped log file in `.o3de/Logs/DRED/`. This log contains all events, draws, barriers, and dispatches leading up to a crash, as well as commands that failed to complete.
+The offending command that likely caused the device removal will be visible after the `===ERROR START===` and before the `===ERROR END===` lines in the log. 
+
+**Example**
+
+An example of such a log displayed in the Windows log viewer is displayed below:
+
+<img src="/images/atom-guide/dev-guide/dred_log_sample.png" alt="DRED Log Sample" style="max-width: 800px"/>
+
+The red lines demarcate the region containing the offending command. Depending on how much work was in-flight on the GPU at the time of the crash, the error region may be larger or smaller. In this example, the offending
+command was a single `DRAWINSTANCED` command issued within the `FullsceenShadowPass`.
+
+{{< note >}}
+The log above had labeled events containing pass names because the engine was compiled with the `-DLY_PIX_ENABLED` CMake configuration flag. For information about integrating the PIX Windows Event Runtime, please
+follow the directions in [this wiki page](https://github.com/o3de/o3de/wiki/CPU-&-GPU-Debugging-and-Profiling-Tools#pix-cpu--gpu-profiling).
+
+For more details about the operation of DRED, consult the DirectX specification on [`WriteBufferImmediate`](https://microsoft.github.io/DirectX-Specs/d3d/D3D12WriteBufferImmediate.html) and [DRED](https://microsoft.github.io/DirectX-Specs/d3d/DeviceRemovedExtendedData.html).
```

Comment:
- Changed "had" to "contains" to make it more clear that "labeled" is an adjective for "events", and not a part of the verb phrase "had labeled". 
- Subsequently changed "contains pass names" to "with pass names", so sentence doesn't have "contain" twice. 
- Added a link to PIX
- Added source names for external tools
- Reformatted text for link to O3DE wiki. 

```suggestion
The log above contains labeled events with pass names because the engine was compiled with the `-DLY_PIX_ENABLED` CMake configuration flag. For information about integrating [Microsoft's PIX Event Runtime for Windows](https://devblogs.microsoft.com/pix/winpixeventruntime/), please follow the directions in the O3DE Wiki page, [CPU& GPU Debugging and Profiling Tools](https://github.com/o3de/o3de/wiki/CPU-&-GPU-Debugging-and-Profiling-Tools#pix-cpu--gpu-profiling).

For more details about the operation of DRED, consult the DirectX specification on [`WriteBufferImmediate`](https://microsoft.github.io/DirectX-Specs/d3d/D3D12WriteBufferImmediate.html) and [DRED](https://microsoft.github.io/DirectX-Specs/d3d/DeviceRemovedExtendedData.html).
```